framework/src/node/bft/api.ts

```diff
@@ -106,15 +101,19 @@ export class BFTAPI extends BaseAPI {
 		};
 	}
 
+	// This function expects to be called after blockBFTInfos has been inserted for the executiong block.
```

Comment:
@janhack Let me know if this assumption is correct 🙏

Reply:
So compared to https://github.com/LiskHQ/lips/blob/main/proposals/lip-0058.md#impliesmaximalprevotes, you changed the function here assuming the block provided as input has already been added to `blockBFTInfos`?

This is fine, but then the input `header` in the function would not be needed and instead the header information from `blockBFTInfos` could be used.

Reply:
yes! Actually, I think this was not related to this PR. When we changed the BFT execution to be in `beforeTransactionsExecute` hook, this needed to be changed as well